ompi/mca/pml/ob1/pml_ob1.c

```diff
@@ -60,32 +61,34 @@
 
 mca_pml_ob1_t mca_pml_ob1 = {
     {
-        mca_pml_ob1_add_procs,
-        mca_pml_ob1_del_procs,
-        mca_pml_ob1_enable,
-        NULL,  /* mca_pml_ob1_progress, */
-        mca_pml_ob1_add_comm,
-        mca_pml_ob1_del_comm,
+        .pml_add_procs      = mca_pml_ob1_add_procs,
+        .pml_del_procs      = mca_pml_ob1_del_procs,
+        .pml_enable         = mca_pml_ob1_enable,
+        .pml_progress       = NULL,  /* mca_pml_ob1_progress, */
```

Comment:
You can safely delete any NULL lines. The language guarantees any uninitialized member is 0ed.

Reply:
<laugh> Brian just asked me to show them for those people who don't remember that it all defaults to NULL 😄 

I figure it causes no harm, and can help with clarity - though I could counter that it raises questions about the fields we _don't_ specify. 🤷‍♂️